protoactor/props.py

```diff
@@ -0,0 +1,102 @@
+from messages import Started
+from context import get_default_receive
+
+def get_default_spawner(name, props, parent):
+    ctx = Context(props.producer, props.supervisor_strategy, props.middleware_chain, parent)

```

Comment:
For now there is no implementation for `Context`. Maybe need to create empty implementation of that.